docs/source/fine-performance-metrics.rst

```diff
@@ -0,0 +1,198 @@
+Fine Performance Metrics
+========================
+.. note::
+
+    This is an experimental feature and may rapidly change without a deprecation cycle.
+
+You may want to investigate where your Dask workload spends the majority of its time;
+not only on which tasks, but also *doing what* while running said tasks.
+*Fine performance metrics* are collected automatically by Dask and attempt to answer
+this question.
+
+In order to observe them, you should simply
+
+#. run your workload end-to-end
+#. open the Dask dashboard (default for a LocalCluster: `<http://localhost:8787>`_)
+#. select ``More...`` -> ``Fine Performance Metrics``
+
+Alternatively, if you're using Jupyter Lab and `dask-labextension
+<https://github.com/dask/dask-labextension>`_, you can just drag the
+``Fine Performance Metrics`` widget onto your Jupyter dashboard.
+
+.. image:: images/fine-performance-metrics/seconds-full.png
+   :alt: Populated Fine Performance Metrics dashboard
+
+The central panel shows, of all the time that the cluster spent doing something, what
+*activities* it spent performing. The most important ones are:
+
+``thread-cpu``
+  CPU time spent by tasks while running on workers
```

Comment:
Maybe add a small comment that this is "good" time

Reply:
done